pkgs/applications/version-management/sr-ht/todo.nix

```diff
@@ -0,0 +1,39 @@
+{ stdenv, fetchgit
+, srht, python3Packages }:
+
+with python3Packages;
+
+buildPythonPackage rec {
+  name = "todo-sr-ht-${version}";
+  version = "0.30.5";
+
+  src = fetchgit {
+    url = "https://git.sr.ht/~sircmpwn/todo.sr.ht";
+    rev = version;
+    sha256 = "197bwhqjg1bkk825420813lqb1ycpn79qja95hpvwwcif396dhn9";
+  };
+
+  patches = [
+    ./env.patch
+  ];
+
+  propagatedBuildInputs = [
+    srht
+    # flask_login
+    alembic
+    pystache
+  ];
+
+  PKGVER = version;
+  SRHT_PATH = "${srht}/${python3Packages.python.sitePackages}/srht";
+
+  # Exception: Config option [meta.sr.ht] origin not found
+  doCheck = false;
```

Comment:
Either fix or explain the error

Reply:
Don't actually know how to fix this but by following the stack trace, it seems like it required a working server? Or at least a configuration file that is present at `/etc/sr.ht/config.ini`.